websites/J/Jet Set Radio Live/presence.ts

```diff
@@ -11,31 +11,37 @@ const presence = new Presence({
     return [Math.floor(startTime / 1000), endTime];
   },
   stationIDMap: { [key: string]: string } = {
-    classic: "Classic",
-    future: "Future",
-    ultraremixes: "Ultra Remixes",
-    ggs: "GG's",
-    noisetanks: "Noise Tanks",
-    poisonjam: "Poison Jam",
-    rapid99: "Rapid 99",
-    loveshockers: "Love Shockers",
-    immortals: "The Immortals",
-    doomriders: "Doom Riders",
-    goldenrhinos: "Golden Rhinos",
-    ganjah: "Ganjah",
-    lofi: "Lo-Fi",
-    siivagunner: "SilvaGunner x JSR",
-    silvagunner: "SilvaGunner x JSR",
-    futuregeneration: "Future Generation",
-    jetmashradio: "Jet Mash Radio",
-    djchidow: "DJ Chidow",
-    hover: "Hover",
-    butterflies: "Butterflies",
-    toejamandearl: "Toe Jam & Earl",
-    ollieking: "Ollie King",
-    crazytaxi: "Crazy Taxi",
-    revolution: "Revolution",
-    endofdays: "End of Days"
+    outerspace: 'Outer Space',
+    classic: 'Classic',
+    future: 'Future',
+    ultraremixes: 'Ultra Remixes',
+    garage: 'The Garage',
+    ggs: 'GG\'s',
+    noisetanks: 'Noise Tanks',
+    poisonjam: 'Poison Jam',
+    rapid99: 'Rapid 99',
+    loveshockers: 'Love Shockers',
+    immortals: 'The Immortals',
+    doomriders: 'Doom Riders',
+    goldenrhinos: 'Golden Rhinos',
+    ganjah: 'Ganjah',
+    lofi: 'Lo-Fi',
+    chiptunes: 'Chiptunes',
+    retroremix: 'Retro Remix',
+    classical: 'Classical Remix',
+    revolution: 'Revolution',
+    endofdays: 'End of Days',
+    silvagunner: 'SilvaGunner x JSR',
+    futuregeneration: 'Future Generation',
+    jetmashradio: 'Jet Mash Radio',
+    crazytaxi: 'Crazy Taxi',
+    ollieking: 'Ollie King',
+    toejamandearl: 'Toe Jam & Earl',
+    hover: 'Hover',
+    butterflies: 'Butterflies',
+    bonafidebloom: 'BonafideBloom',
+    djchidow: 'DJ Chidow',
+    verafx: 'VeraFX'
```

Comment:
Single quotes won't pass ESLint, you need to use double quotes (`"`) instead
```suggestion
    outerspace: "Outer Space",
    classic: "Classic",
    future: "Future",
    ultraremixes: "Ultra Remixes",
    garage: "The Garage",
    ggs: "GG\'s",
    noisetanks: "Noise Tanks",
    poisonjam: "Poison Jam",
    rapid99: "Rapid 99",
    loveshockers: "Love Shockers",
    immortals: "The Immortals",
    doomriders: "Doom Riders",
    goldenrhinos: "Golden Rhinos",
    ganjah: "Ganjah",
    lofi: "Lo-Fi",
    chiptunes: "Chiptunes",
    retroremix: "Retro Remix",
    classical: "Classical Remix",
    revolution: "Revolution",
    endofdays: "End of Days",
    silvagunner: "SilvaGunner x JSR",
    futuregeneration: "Future Generation",
    jetmashradio: "Jet Mash Radio",
    crazytaxi: "Crazy Taxi",
    ollieking: "Ollie King",
    toejamandearl: "Toe Jam & Earl",
    hover: "Hover",
    butterflies: "Butterflies",
    bonafidebloom: "BonafideBloom",
    djchidow: "DJ Chidow",
    verafx: "VeraFX"
```